velox/functions/sparksql/aggregates/tests/BloomFilterAggAggregateTest.cpp

```diff
@@ -71,14 +71,15 @@ TEST_F(BloomFilterAggAggregateTest, emptyInput) {
   testAggregations(vectors, {}, {"bloom_filter_agg(c0, 5, 64)"}, expected);
 }
 
-TEST_F(BloomFilterAggAggregateTest, nullBloomFilter) {
-  auto vectors = {makeRowVector({makeAllNullFlatVector<int64_t>(2)})};
-  auto expectedFake = {makeRowVector(
-      {makeNullableFlatVector<StringView>({std::nullopt}, VARBINARY())})};
-  VELOX_ASSERT_THROW(
-      testAggregations(
-          vectors, {}, {"bloom_filter_agg(c0, 5, 64)"}, expectedFake),
-      "First argument of bloom_filter_agg cannot be null");
+TEST_F(BloomFilterAggAggregateTest, withNullInput) {
+  // The first 10 values range from 0 to 9, and the last 10 values are nulls
+  auto vectors = {makeRowVector({makeFlatVector<int64_t>(
+      20,
+      [](vector_size_t row) { return row % 9; },
+      [](vector_size_t row) { return row > 9; })})};
+  // The expected value is the serialized bloom filter
+  auto expected = {makeRowVector({getSerializedBloomFilter(4)})};
```

Comment:
Can we also test all-null input? I remember seeing that it had a somewhat unexpected (non-NULL) result. Let's document what that result is.